docs/maintain/maintain-wss.md

```diff
@@ -127,20 +151,31 @@ server {
 }
 ```
 
-Restart nginx after setting this up: `sudo service nginx restart`.
+### Apache2
 
-## Importing the Certificate
+Apache2 is slightly more computationally intensive when compared to nginx but also provides more configurable options. You can run it in different modes such as prefork, worker or event. In this example, we use
```

Comment:
> Apache2 is slightly more computationally intensive when compared to nginx but also provides more configurable options

I think we should refrain from doing any judgement on Apache vs nginx but ideally provide the 2 config examples.

Here is a snipped based on our internal config. Could you kindly double check it on your side and add it.

```
server {
  listen                443 ssl;
  server_name           YOUR_DOMAIN;
# for letsencrypt
  ssl_certificate /etc/letsencrypt/live/YOUR_DOMAIN/fullchain.pem;
  ssl_certificate_key /etc/letsencrypt/live/YOUR_DOMAIN/privkey.pem;
# for self-signed
  ssl_certificate /etc/ssl/certs/selfsigned.crt;
  ssl_certificate_key /etc/ssl/private/selfsigned.key;
  location / {
    limit_req zone=zone burst=5;
    proxy_buffers 16 4k;
    proxy_buffer_size 2k;
    proxy_pass ws://localhost:9944;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header Host $host;
   }
}
```

Reply:
I made some more changes; I find the very specific ssl instruction making it more confusing. I think that somebody who sets up a rpc node will know how to setup a ssl enabled host and the specific rpc/proxypass config is the relevant part. Let me know if you disagree and think it needs to be more explicit.